Index: code-editor-src/extensions/shared.webpack.config.mjs
===================================================================
--- code-editor-src.orig/extensions/shared.webpack.config.mjs
+++ code-editor-src/extensions/shared.webpack.config.mjs
@@ -17,6 +17,7 @@ const require = createRequire(import.met
 const tsLoaderOptions = {
 	compilerOptions: {
 		'sourceMap': true,
+		'rootDir': '.',
 	},
 	onlyCompileBundledFiles: true,
 };
